src/main/java/pl/isa/biblioteka/user/LogUser.java

```diff
@@ -0,0 +1,46 @@
+package pl.isa.biblioteka.user;
+
+import pl.isa.biblioteka.books.Book;
+import pl.isa.biblioteka.menu.LibrarianMenu;
+import pl.isa.biblioteka.menu.UserMenu;
+
+import java.util.List;
+import java.util.Scanner;
+
+public class LogUser {
+    private static Scanner scanner = new Scanner(System.in);
+    public static Person logPerson;
+    public LogUser() {
```

Comment:
Jak nie ma innych konstruktorów, ani ten nic nie robi, to wywalić

Reply:
poprawione